### DISCUSSION TITLE
Enhance the pick many text based prompt

### DISCUSSION THREAD

**Discussion by asmodeus812** [mfussenegger/nvim-jdtls#392] [Ideas]
Hi, i would like to ask if there is some better approach to be taken as far as the particular feature is concerned ? The pick many is quite primitive and somewhat annoying to use (overlaps the buffer with multiple selection) Why not rethink the ui design a bit to be more user-friendly. My biggest gripe is there is no select all for example, which for things like to_string or equals and hashCode are pretty much the most common use case. Thanks !

**Comment by mfussenegger** [comment]
See https://github.com/mfussenegger/nvim-jdtls/issues/358
